$NetBSD: patch-Makefile,v 1.5 2023/01/06 17:42:01 jperkin Exp $

Exclude LDFLAGS, this is not C.
Use find(1) in a more portable way.

--- Makefile.orig	2022-12-29 19:08:57.000000000 +0000
+++ Makefile
@@ -96,7 +96,7 @@ else
 	endif
 endif
 
-LDFLAGS := $(LDFLAGS) -X "main.MakeVersion=$(MAKE_VERSION)" -X "main.Version=$(GITEA_VERSION)" -X "main.Tags=$(TAGS)"
+LDFLAGS := -X "main.MakeVersion=$(MAKE_VERSION)" -X "main.Version=$(GITEA_VERSION)" -X "main.Tags=$(TAGS)"
 
 LINUX_ARCHS ?= linux/amd64,linux/386,linux/arm-5,linux/arm-6,linux/arm64
 
@@ -133,7 +133,7 @@ GO_DIRS := cmd tests models modules rout
 WEB_DIRS := web_src/js web_src/less
 
 GO_SOURCES := $(wildcard *.go)
-GO_SOURCES += $(shell find $(GO_DIRS) -type f -name "*.go" -not -path modules/options/bindata.go -not -path modules/public/bindata.go -not -path modules/templates/bindata.go)
+GO_SOURCES += $(shell find $(GO_DIRS) -type f -name "*.go" -and ! -path modules/options/bindata.go -and ! -path modules/public/bindata.go -and ! -path modules/templates/bindata.go)
 GO_SOURCES += $(GENERATED_GO_DEST)
 GO_SOURCES_NO_BINDATA := $(GO_SOURCES)
 
